script.sh

```diff
@@ -44,29 +44,57 @@ echo -e "\n\n"
 bold "-------------------------------"
 echo -e "\n\n"
 
+# Return value if the file to download already exists
+FILE_ALREADY_EXISTS=100
+
 # Custom wget command, with continue and quiet options, as well path specification
 function _wget {
   wget -c ${1} -P videos --quiet
 }
+# Custom aria2 command, download each file using 16 parallel connections
+# _aria2c URL filename
+function _aria2c {
+  # if the file to download already exists and there isn't the corresponding
+  # aria2 file, so the download is completed
+  if [ -e videos/${2} ] && [ ! -e videos/${2}.aria2 ]; then
+    return $FILE_ALREADY_EXISTS
+  fi
+  aria2c -s16 --dir videos --quiet --auto-file-renaming=false ${1}
+}
+
+function download {
+  # if there is aria2, use it!
+  if [ -n "`which aria2c`" ]; then
```

Comment:
the backtic syntax is deprecated...

can you change it to `"$(which aria2c)"`?